### ISSUE TITLE
Question about training code

### ISSUE THREAD

**Issue by LILILILILIYIRAN** [autonomousvision/plant#14]
Hello, I get a strange bug in the code when I run the training code . It mention me that "targets" can't be returned in the model.py. Should I delete the "return tragets"? Or give "targets=None"?
Thanks.

294            return logits, targets, pred_wp, attn_map

**Comment by LILILILILIYIRAN** [comment]
Hello, I get a strange bug in the code when I run the training code . It mention me that "targets" can't be return in the model.py. Should I delete the "return tragets"? Or give "targets=None"?
Thanks.

**Comment by LILILILILIYIRAN** [comment]
I think you can delete targets in 294 lines or add the targets=None after the logits=None. But in this case you need to fix the lit_model.py. This is my solution.
